index.d.ts

```diff
@@ -339,4 +343,41 @@ declare module 'binance-api-node' {
         isOrderWorking: boolean;
         isBuyerMaker: boolean;
     }
+
+    interface MyTrade {
+        id: number;
+        orderId: number;
+        price: string;
+        qty: string;
+        commission: string;
+        commissionAsset: string;
+        time: number;
+        isBuyer: boolean;
+        isMaker: boolean;
+        isBestMatch: boolean;
+    }
+
+    interface QueryOrderResult {

```

Comment:
Can `interface QueryOrderResult` and `interface Order` be merged?

Reply:
No. Order uses `transactTime` instead of `time` and provides some additional fields.